ansible/vars/dev/dev_public.yml

```diff
@@ -127,6 +127,9 @@ backup_blobdb: True
 postgres_s3: False
 nadir_hour: 18
 
+nofile_limit: 65536
+nginx_worker_rlimit_nofile: "{{ nofile_limit }}"
```

Comment:
where are these used?

Reply:
nofile_limit is the general max files open and nginx_worker_rlimit_nofile only for nginx max files open,  maybe this one is redudant?

Reply:
OK, I thought you had introduced these in this PR but I see they are already in use.